src/Generator/TraitUsageGenerator.php

```diff
@@ -11,6 +11,12 @@
 use Reflection;
 use ReflectionMethod;
 
+/**
+ * Class TraitUsageGenerator
+ *
+ * @deprecated Deprecated in 2.7. Use UsageGenerator instead
```

Comment:
SNAFU then :-(

IMO, the `TraitUsageGenerator` should be used for traits, and maybe the other one can be `ImportGenerator` or such...